### PR TITLE
Mention that ZEIT is the author of Next.js

### PR DESCRIPTION
**Pull request by lfades**
Updated descriptions in the site

### CODE REVIEW

**Comment by zeit-team** [comment]
#### 📝Changed routes:


| [**/blog**](https://next-site-git-next-by-zeit.zeit.now.sh/blog) | [**…basics/getting-started**](https://next-site-git-next-by-zeit.zeit.now.sh/learn/basics/getting-started) |
|:-:|:-:|
|<a href="https://next-site-git-next-by-zeit.zeit.now.sh/blog">Screenshot of /blog</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/e0V_DDgEbpB82s_KLzl1VhrNsik_">(view full size)</a> |<a href="https://next-site-git-next-by-zeit.zeit.now.sh/learn/basics/getting-started">Screenshot of /learn/basics/getting-started</a><br /><sup><a href="https://images.weserv.nl?url=https://microlink.nyc3.digitaloceanspaces.com/m8qxTURFx86OnDEmk_tXSNQJw5o_">(view full size)</a> |







Commit d7cf794d035bb9254c38712ab592230e90973c28 (https://next-site-git-next-by-zeit.zeit.now.sh).

File: pages/blog/index.js [hunk 1]
```diff
@@ -62,7 +62,15 @@ export default () => (
     <Page title="Blog | Next.js">
       <Screen offset={64 + 400}>
         <Container padding wide>
-          <SectionHeader title="Blog" />
+          <SectionHeader
+            title="Blog"
+            description={
+              <span>
+                The latest news about Next.js <br className="display-mobile" />
+                by the ZEIT team
```
Comment (pacocoursey) [inline pages/blog/index.js]:
```suggestion
                from the ZEIT team
```

File: package.json [hunk 2]
```diff
@@ -1,9 +1,9 @@
 {
   "name": "next-site",
-  "description": "Next.js Official Homepage",
+  "description": "Next.js by ZEIT Official Homepage",
```
Comment (timneutkens) [inline package.json]:
```suggestion
  "description": "Next.js by ZEIT: Official Homepage",
```

File: components/home/intro.js [hunk 3]
```diff
@@ -235,6 +235,9 @@ export default class extends React.PureComponent {
                 <div>
                   <Button href="https://github.com/zeit/next.js">GitHub</Button>
                 </div>
+                <div>
+                  <Button href="https://zeit.co">ZEIT</Button>
```
Comment (timneutkens) [inline components/home/intro.js]: I think this button is non-actionable, we should probably wait on @evilrabbit before making this change

File: README.md [hunk 4]
```diff
@@ -22,5 +22,5 @@ We really appreciate any contribution.
 
 ## Related Repositories
 
-- [Next.js](https://github.com/zeit/next.js)
-- [Next.js Plugins](https://github.com/zeit/next-plugins)
+- [zeit/next.js](https://github.com/zeit/next.js)
+- [zeit/next-plugins](https://github.com/zeit/next-plugins)
```
Comment (timneutkens) [inline README.md]:
```suggestion
```

File: README.md [hunk 5]
```diff
@@ -22,5 +22,5 @@ We really appreciate any contribution.
 
 ## Related Repositories
 
-- [Next.js](https://github.com/zeit/next.js)
-- [Next.js Plugins](https://github.com/zeit/next-plugins)
+- [zeit/next.js](https://github.com/zeit/next.js)
```
Comment (timneutkens) [inline README.md]:
```suggestion
- [Next.js GitHub repository](https://github.com/zeit/next.js)
```

File: components/page.js [hunk 6]
```diff
@@ -12,7 +12,7 @@ function Page({ title, description, children }) {
   return (
     <div>
       <Head>
-        <title>{title || 'Next.js - The React Framework'}</title>
+        <title>{title || 'Next.js by ZEIT - The React Framework.'}</title>
```
Comment (Timer) [inline components/page.js]:
```suggestion
        <title>{title || 'Next.js by ZEIT - The React Framework'}</title>
```

File: components/showcase/title.js [hunk 7]
```diff
@@ -33,9 +32,8 @@ export default ({ height }) => {
           title="Showcase"
           description={
             <span>
-              Meet hundreds of beautiful websites{' '}
-              <br className="display-mobile" />
-              powered by Next.js
+              Meet hundreds of beautiful websites <br className="display-mobile" />
+              powered by ZEIT's Next.js
```
Comment (chibicode) [inline components/showcase/title.js]:
```suggestion
              built with Next.js by ZEIT
```

**Comment by chibicode** [comment]
This branch was merged into https://github.com/zeit/next-site/pull/635 and updated using constants / components. Closing in favor of it.

**Pull request closed without merging**